packages/blob-index/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@web3-storage/blob-index",
```

Comment:
The blob index is a subset of the sharded DAG index should this be named something different?

Suggestion, call it `@web3-storage/index`? Then have exports for `@web3-storage/index/dag-sharded` and `@web3-storage/index/blob`?

Reply:
Well, it is for indexing blobs. I mean, a DAG is what is being stored, but the DAG 's chunks that are stored in one or more blobs is what is being indexed. Whereas if a DAG was stored in some other construct, say bit-bucket, then it would be a bit-bucket-index, even though a sharded-dag is still being indexed. That's what I was thinking.

If you see this same indexing scheme being used universally, whether the DAG is stored in a blob, bit-bucket, etc., then yes, let's call it an "index" as you suggest.

Reply:
Ok makes sense lets go with it!